prometheus/internal/difflib.go

```diff
@@ -46,6 +47,97 @@ func calculateRatio(matches, length int) float64 {
 	return 1.0
 }
 
+var (
```

Comment:
I wouldn't put this in difflib.go because that file is really a (possibly modified) copy of https://github.com/pmezard/go-difflib/blob/master/difflib/difflib.go . Better to use a separate file for the AlmostEqual code.